plugins/authentication/src/OAuthModel/model.tsx

```diff
@@ -0,0 +1,475 @@
+import { ConfigurationReference, getConf } from '@jbrowse/core/configuration'
+import { InternetAccount } from '@jbrowse/core/pluggableElementTypes/models'
+import PluginManager from '@jbrowse/core/PluginManager'
+import { isElectron } from '@jbrowse/core/util'
+
+import { OAuthInternetAccountConfigModel } from './configSchema'
+import crypto from 'crypto'
+import { Instance, types } from 'mobx-state-tree'
+import { UriLocation } from '@jbrowse/core/util/types'
+import { RemoteFile } from 'generic-filehandle'
+
+interface OAuthData {
+  client_id: string
+  redirect_uri: string
+  response_type: 'token' | 'code'
+  scope?: string
+  code_challenge?: string
+  code_challenge_method?: string
+  token_access_type?: string
+}
+
+const inWebWorker = typeof sessionStorage === 'undefined'
+
+const stateModelFactory = (
+  pluginManager: PluginManager,
+  configSchema: OAuthInternetAccountConfigModel,
+) => {
```

Comment:
The pluginManger isn't used, so it can be removed.

```suggestion
const stateModelFactory = (configSchema: OAuthInternetAccountConfigModel) => {
```